src/View/Helper.php

```diff
@@ -144,11 +144,11 @@ protected function _confirm(string $okCode, string $cancelCode): string
      * Adds the given class to the element options
      *
      * @param array $options Array options/attributes to add a class to
-     * @param string|null $class The class name being added.
-     * @param string $key the key to use for class.
+     * @param string $class The class name being added.
+     * @param string $key the key to use for class. Defaults to `'class'`.
      * @return array Array of options with $key set.
      */
-    public function addClass(array $options = [], ?string $class = null, string $key = 'class'): array
+    public function addClass(array $options, string $class, string $key = 'class'): array
```

Comment:
I removed the default `null` for class. Doesn't make sense for a method meant to add class accepting `null` value for class.